File: DIP/DIP_Lab_406/Lab/353_lab_04/Lab_04_python/PixelSkipping.py
import cv2
import numpy as np
from matplotlib import pyplot as plt
image = cv2.imread('nature.jpeg')
image_rgb = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)
skip_factor = 3  
skipped_image = image_rgb[::skip_factor, ::skip_factor]
plt.figure(figsize=(12, 6))
plt.subplot(1, 2, 1)
plt.imshow(image_rgb)
plt.title('Original Image')
plt.axis('off')
plt.subplot(1, 2, 2)
plt.imshow(skipped_image)
plt.title('Pixel Skipped Image')
plt.axis('off')
plt.show()

File: DIP/DIP_Lab_406/Lab/353_lab_04/Lab-04_matlab/PixelSkipping.m
image = imread('nature.jpeg');
if size(image, 3) == 3
    grayImage = rgb2gray(image);
else
    grayImage = image;
end
skipFactor = 4;   
skippedImage = grayImage(1:skipFactor:end, 1:skipFactor:end);
figure;
subplot(1, 2, 1);
imshow(grayImage);
title('Original Image');
subplot(1, 2, 2);
imshow(skippedImage);
title('Pixel Skipped Image');
